spec/02-integration/04-admin_api/06-certificates_routes_spec.lua

```diff
@@ -424,6 +424,7 @@ describe("Admin API: #" .. strategy, function()
       it("upserts if found", function()
         local certificate = add_certificate()
 
+        ngx.sleep(1)
```

Comment:
May I know what's the purpose of this line?

Reply:
Give some time to validate the updated_at will be updated